src/core_modules/capture-core/components/Pages/ViewEvent/EventDetailsSection/ViewEventDataEntry/viewEventDataEntry.epics.js

```diff
@@ -50,15 +51,17 @@ export const loadViewEventDataEntryEpic = (action$: InputObservable, store: Redu
             const viewEventPage = state.viewEventPage || {};
             return viewEventPage.eventId === eventId && !viewEventPage.showEditEvent;
         }),
-        switchMap(async (action) => {
+        switchMap((action) => {
```

Comment:
here I removed the async and instead I did `from(Promise)` and inside the pipe I dispatched the batch of actions

Reply:
Looks ok 👍